### ISSUE TITLE
Is there a guide to add a native app?

### ISSUE THREAD

**Issue by mansoursaid001** [beardedspice/beardedspice#861]
I have started using anghami's native app and I would like to add its support. I have cloned the repo and went through it but couldn't mimic the already existing integrations.

It would be great if there is a guide or just a basic outline to what I need to do and I will discover the rest.

**Comment by nSimonFR** [comment]
Yes, https://github.com/beardedspice/beardedspice/wiki/Native-App-Developers'-Guide
